src/agent/main.cpp

```diff
@@ -88,6 +89,7 @@ static const struct option kOptions[] = {
     {"help", no_argument, nullptr, OTBR_OPT_HELP},
     {"thread-ifname", required_argument, nullptr, OTBR_OPT_INTERFACE_NAME},
     {"verbose", no_argument, nullptr, OTBR_OPT_VERBOSE},
+    {"syslog-disable", no_argument, nullptr, OTBR_OPT_SYSLOG_DISABLE},
```

Comment:
nit: I'm inclined to making it a positive flag (i.e. enabling syslog) instead of a negative one. In this way it makes more sense to abbreviate it as `-s`.

Reply:
I felt the same, but didn't want to change existing functionality. Folks would have to go add `-s` to their invocations if they were relying on syslog presently. If you're okay with it being a "breaking" change I can do it, but figured it was best to avoid.

Reply:
I also agree to keep the existing behavior. But is it possible to make the flag default to be true?

That is, you need to pass in something like `-s false` to disable syslog. If you don't pass in `-s` option it will enable syslog.

I don't have a strong preference on this. For the simplicity of implementation I'm fine with the `syslog-disable` flag.

Reply:
I think in the interests of back compat as well as avoiding a (I feel) similarly clunky flag, that we ought to stick with this implementation.